stubs/ibm-db/ibm_db.pyi

```diff
@@ -0,0 +1,233 @@
+def columns(
+    connection: IBM_DBConnection,
+    qualifier: str | None = ...,
+    schema: str | None = ...,
+    table_name: str | None = ...,
+    column_name: str | None = ...,
+) -> IBM_DBStatement: ...
+def commit(connection: IBM_DBConnection) -> bool: ...
+def conn_error(connection: IBM_DBConnection | None = ...) -> str: ...
+def conn_errormsg(connection: IBM_DBConnection | None = ...) -> str: ...
+def conn_warn(connection: IBM_DBConnection | None = ...) -> str: ...
+def connect(
+    database: str, user: str, password: str, options: dict[int, int | str] | None = ..., replace_quoted_literal: int = ...
+) -> IBM_DBConnection | None: ...
+def createdb(connection: IBM_DBConnection, dbName: str, codeSet: str | None = ..., mode: str | None = ...) -> bool: ...
+def createdbNX(connection: IBM_DBConnection, dbName: str, codeSet: str | None = ..., mode: str | None = ...) -> bool: ...
+def cursor_type(stmt: IBM_DBStatement) -> int: ...
+def dropdb(connection: IBM_DBConnection, dbName: str) -> bool: ...
+def exec_immediate(connection: IBM_DBConnection, statement: str, options: dict[int, int]) -> IBM_DBStatement | bool: ...
+def execute(stmt: IBM_DBStatement, parameters: tuple[object, ...] | None = ...) -> bool: ...
+def execute_many(stmt: IBM_DBStatement, seq_of_parameters: tuple[object, ...]) -> int | None: ...
+def fetch_assoc(stmt: IBM_DBStatement, row_number: int | None = ...) -> dict[str, object] | bool: ...
+def fetch_both(stmt: IBM_DBStatement, row_number: int | None = ...) -> dict[int | str, object] | bool: ...
+def fetch_row(stmt: IBM_DBStatement, row_number: int | None = ...) -> bool: ...
+def fetch_tuple(stmt: IBM_DBStatement, row_number: int | None = ...) -> tuple[object, ...]: ...
+def field_display_size(stmt: IBM_DBStatement, column: int | str) -> int | bool: ...
+def field_name(stmt: IBM_DBStatement, column: int | str) -> str | bool: ...
+def field_nullable(stmt: IBM_DBStatement, column: int | str) -> bool: ...
+def field_num(stmt: IBM_DBStatement, column: int | str) -> str | bool: ...
+def field_precision(stmt: IBM_DBStatement, column: int | str) -> int | bool: ...
+def field_scale(stmt: IBM_DBStatement, column: int | str) -> int | bool: ...
+def field_type(stmt: IBM_DBStatement, column: int | str) -> str | bool: ...
+def field_width(stmt: IBM_DBStatement, column: int | str) -> int | bool: ...
+def foreign_keys(
+    connection: IBM_DBConnection,
+    pk_qualifier: str,
+    pk_schema: str,
+    pk_table_name: str,
+    fk_qualifier: str,
+    fk_schema: str,
+    fk_table_name: str,
+) -> IBM_DBStatement: ...
+def free_result(stmt: IBM_DBStatement) -> bool: ...
+def free_stmt(stmt: IBM_DBStatement) -> bool: ...
+def get_db_info(connection: IBM_DBConnection, option: int) -> str | bool: ...
+def get_last_serial_value(stmt: IBM_DBStatement) -> str | bool: ...
+def get_num_result(stmt: IBM_DBStatement) -> int | bool: ...
+def get_option(resc: IBM_DBConnection | IBM_DBStatement, options: int, type: int) -> object: ...
```

Comment:
```suggestion
def get_option(resc: IBM_DBConnection | IBM_DBStatement, options: int, type: int) -> Any: ...
```
As far as I can tell it returns int, str, or bool depending on the type. We prefer Any as the return type here since union return types are inconvenient.